### PR TITLE
[REG-19][Feature]Overview Page Layout

### PR DESCRIPTION
**Pull request by chrisseto**
## Purpose

* Implement and add the registries-navbar
* Add basic layout components and styles of the overview page

## Side Effects

None, because feature flags

## Feature Flags

* `ember_registries_new_style` - When enabled the new navbar will be used and styling from the overview mocks up will be applied to all of registries

## QA Notes

Only the navbar is QA'able here the overview page itself is disabled in production builds.

## Ticket

https://openscience.atlassian.net/browse/EMB-

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: lib/registries/addon/components/registries-navbar/template.hbs [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<Navbar @dark={{true}} as |nav|>
+    <nav.container>
+        {{!-- Left Side (Band, Service Dropdown) --}}
+        <nav.section data-test-left>
+
+            {{#global-link-to data-test-brand 'home' (html-attributes aria-label=(t 'navbar.go_home'))}}
```
[the remark this answers is not in the document]
Comment (chrisseto) [reply lib/registries/addon/components/registries-navbar/template.hbs]: 👍

File: lib/registries/addon/application/controller.ts [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
 
 export default class Application extends Controller {
+    @service features!: Features;
+
     activeService = OSFService.REGISTRIES;
     searchRoute = 'registries.discover';
     supportRoute = 'http://help.osf.io/m/registrations/';
+
+    @computed(`features.${camelize(newStyle)}`)
+    get newStyleEnabled(): boolean {
```
[the remark this answers is not in the document]
Comment (chrisseto) [reply lib/registries/addon/application/controller.ts]: 👍

File: lib/registries/addon/components/registries-navbar/template.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrisseto) [reply lib/registries/addon/components/registries-navbar/template.hbs]: Actually it looks like `global-link-to` (and `link-to` itself) does some weird stuff with arguments and positional parameters. 👎

File: lib/registries/addon/components/gutters/styles.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        $border-side: 'left';
+    } @else {
+        $modifier: -1;
+        $border-side: 'right';
+    }
+
+    border-#{$border-side}: $alto 1px solid;
+
+
+    &[data-gutter-mode='page'] {
```
[the remark this answers is not in the document]
Comment (chrisseto) [reply lib/registries/addon/components/gutters/styles.scss]: I agree on passing data down with a yield but I think the data attributes make the styling a lot more clear and come testing time there's no need for data-test selectors.

[the remark this answers is not in the document]
Comment (aaxelb) [reply lib/registries/addon/components/gutters/styles.scss]: hmmm my reflexive response is strong disagreement but now I'm stewing in uncertainty.

http://amcss.github.io/

**Pull request merged**